doc/USING.rst

```diff
@@ -12,7 +12,7 @@ Inference using a DeepSpeech pre-trained model can be done with a client/languag
 Running ``deepspeech`` might, see below, require some runtime dependencies to be already installed on your system:
 
 
-* ``sox`` - The Python and Node.JS clients use SoX to resample files to 16kHz.
+* ``sox`` - The Python and Node.JS clients use SoX to resample files to 16kHz. On MacOS, the Command-Line client will also require SoX to be installed.
 * ``libgomp1`` - libsox (statically linked into the clients) depends on OpenMP. Some people have had to install this manually.
```

Comment:
@madprogramer Maybe this is where it might be improved, adding a `libsox` line mentionning that on macOS (and maybe windows?) this is a needed dep for the C++ client ?

Reply:
However a sox installation (via `brew install sox`) is by far the most straightforward way to install libsox, which is why I added it there. 

This is an issue unique to the MacOS Command-Line client, since as far as I can tell it's statically linked into the Windows and Linux clients.

Perhaps it might be better to change 
```
* ``libgomp1`` - libsox (statically linked into the clients) depends on OpenMP. Some people have had to install this manually.
```
to 
```
* ``libgomp1`` - libsox (statically linked into the Windows and Linux clients) depends on OpenMP. Some people have had to install this manually.
``` 
then?

Reply:
> However a sox installation (via `brew install sox`) is by far the most straightforward way to install libsox, which is why I added it there.

I have no macOS, and the experience I gathered on doing CI makes me hate brew. My point is, we technically dont need the binary, so we should name the `libsox` dependency explicitely. We can always word it to say that you can install it using this.



> Perhaps it might be better to change

If you feel that it is clear, why not.

Reply:
@reuben Do you think it's enough like that?

Reply:
I agree with @madprogramer, `brew install sox` is the simplest way to install `libsox` on macOS, so we should just say that. I'd say we should go even further and explicitly break down this section into several sections with copy-pasteable lines for Ubuntu/macOS/Windows/whatever.